src/main/java/kohn/maze/gui/Cell.java

```diff
@@ -0,0 +1,36 @@
+package kohn.maze.gui;
+
+import java.util.ArrayList;
+
+public class Cell {
```

Comment:
The point of this is not to copy and paste the command line code, it's to be able to use that code in writing your GUI